src/components/list/list.scss

```diff
@@ -0,0 +1,9 @@
+.list-container{
+    border: 1px gray solid;
```

Comment:
Border is not the same 
expected 

<img width="950" alt="Screen Shot 2022-09-26 at 9 54 48" src="https://user-images.githubusercontent.com/6474358/192309922-9a6fdb78-245e-4245-a314-4bc0f46df646.png">

actual 
![image](https://user-images.githubusercontent.com/6474358/192309993-7c843de4-2861-4497-971b-34a518765bdb.png)